v2/parser/parse.go

```diff
@@ -486,7 +487,7 @@ func (p *Parser) addPkgToUniverse(pkg *packages.Package, u *types.Universe) erro
 	for _, n := range s.Names() {
 		switch obj := s.Lookup(n).(type) {
 		case *gotypes.TypeName:
-			t := p.walkType(*u, nil, obj.Type())
+			t := p.walkType(*u, nil, unalias(obj.Type()))
```

Comment:
default behavior prior to go1.23 resolved top-level aliases to their underlying types ... this preserves that